src/app/organizations/collection/collection.component.html

```diff
@@ -37,25 +37,19 @@
         <mat-icon>info</mat-icon> This collection is part of an organization that has been rejected by a curator. Members can re-request
         approval on the <a [routerLink]="'/accounts'" style="text-decoration: underline">accounts page</a>.
       </mat-card>
-      <mat-card fxFlex class="my-3">
-        <div fxLayout="row" fxLayout.lt-sm="column" fxLayoutGap="10px">
-          <div>
-            <img [src]="(gravatarUrl$ | async) || '../../../assets/images/dockstore/PlaceholderLC.png'" alt="placeholder" class="orgLogo" />
+      <div fxFlex class="my-3">
+        <div fxLayout="row" fxLayout.lt-sm="column" fxLayoutGap="2rem">
+          <div *ngIf="gravatarUrl$ | async" fxLayoutAlign="center center" fxHide.lt-sm>
+            <img [src]="gravatarUrl$ | async" class="orgLogo" alt="organization logo" />
           </div>
           <div>
-            <mat-card-header>
-              <mat-card-title>
-                <h1>
-                  <a [routerLink]="['/organizations/', organization.name]"
-                    ><span [matTooltip]="organization?.name">{{ organization.displayName }}</span></a
-                  >
-                  / <span [matTooltip]="collection?.name">{{ collection.displayName }}</span>
-                </h1>
-              </mat-card-title>
-              <mat-card-subtitle>
-                {{ collection.topic }}
-              </mat-card-subtitle>
-            </mat-card-header>
+            <h2>
+              <a [routerLink]="['/organizations/', organization.name]">
+                <span [matTooltip]="organization?.name" class="org-name">{{ organization.displayName }}</span></a
```

Comment:
Was like this and I'm not sure what the alternative is, but it's weird to me to just have the name as the tooltip for a display name. I guess just leave it. :)